src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Form/combinations_form_theme.html.twig

```diff
@@ -31,11 +31,14 @@
 {%- endblock -%}
 
 {%- block combination_item_row -%}
-  <tr>
+  <tr id="combination-list-row-{{ form.vars.name }}">
```

Comment:
It doesn't seem to be used for now, but it's a good idea!

Reply:
yeah I was using it to target specific combination image, but ended up not using it because we search for tag in each $row template (we don't have the tr rendered yet at all). Anyway even if we don't use this, I would like to leave this one, it seems usefull to identify the table row later (like for module devs or so)

Reply:
Yep I agree